test/Bootstrapper.FunctionalTests/BootstrapperTests.cs

```diff
@@ -114,7 +114,7 @@ public void BootstrapperShowsVersionAndReturnsZeroExitCodeWhenVersionOptionWasGi
 
         [Theory]
         [MemberData(nameof(RuntimeComponents))]
-        public void BootstrapperInvokesApplicationHostWithInferredAppBase_ProjectDirAsArgument(string flavor, string os, string architecture)
+        public void BootstrapperInvokesApplicationHostWithExplicitAppBase(string flavor, string os, string architecture)
```

Comment:
We need to cover more cases here:

- Current path:
```
dnx run
```
- Explcit `--appbase`:
```
dnx --appbase {path} Microsoft.Framework.ApplicationHost
```

- Specified path implicit ApplicationHost

```
dnx --project {path} run
```